torch/quantization/observer.py

```diff
@@ -234,8 +247,8 @@ class PerChannelMinMaxObserver(_ObserverBase):
     def __init__(self, ch_axis=0, **kwargs):
         super(PerChannelMinMaxObserver, self).__init__(**kwargs)
         self.ch_axis = ch_axis
-        self.min_vals = None
-        self.max_vals = None
+        self.register_buffer('min_vals', None)
```

Comment:
maybe just register empty tensor? (torch.tensor([])). Then you don't need to override _load_from_state_dict?

Reply:
Will try that

Reply:
Doesnt work, so sticking with current approach